File: Assets/script/GameController.js
#pragma strict


var  IsGameOver : boolean = false;
static var instance : GameController;

function Awake(){
	if(instance == null){
		instance = this;
	}
	if(instance != this){
		Destroy(gameObject);
	}
}
function Start () {

}

function Update () {

	if(IsGameOver && Input.GetKeyDown(KeyCode.Space)){
		Application.LoadLevel(Application.loadedLevel);
	}	
}

File: Assets/script/GameController.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class GameController : MonoBehaviour {

	public static GameController instance;
	public bool gameOver = false;

	void Awake(){
		if (instance == null) {
			instance = this;
		}else{
			Destroy (gameObject);
		}
	}

	// Use this for initialization
	void Start () {
		
	}
	
	// Update is called once per frame
	void Update () {

		if (gameOver  && Input.GetKeyDown (KeyCode.Space)) {
			SceneManager.LoadScene (SceneManager.GetActiveScene().buildIndex);		
		}
	}



}
